rosidl_typesupport_opensplice_cpp/bin/rosidl_typesupport_opensplice_cpp

```diff
@@ -59,9 +60,18 @@ def main(argv=sys.argv[1:]):
         return 1
     if rc:
         return rc
+    dds_interface_files = []
+    if not os.path.exists(args.dds_interface_files_file):
+        print("File does not exist: {0}".format(args.dds_interface_files_file), file=sys.stderr)
+        return 1
```

Comment:
Should we let `argparse` only accept a parameter which is an existing file instead?